e2e-tests/deployers/template/cloudflare/src/mastra/tools/weather-tool.ts

```
@@ -40,6 +40,84 @@ export const weatherTool = createTool({
+  current: {
+    time: string;
+    temperature_2m: number;
+    apparent_temperature: number;
+    relative_humidity_2m: number;
+    wind_speed_10m: number;
+    wind_gusts_10m: number;
+    weather_code: number;
+  };
+}
+
+export const weatherTool = createTool({
```

Biome (2.1.2)

[error] Shouldn't redeclare 'weatherTool'. Consider to delete it or rename it.

'weatherTool' is defined here:


(lint/suspicious/noRedeclare)